src/app/modules/common/dialog/confirm-dialog/confirm-dialog.component.html

```diff
@@ -38,6 +38,7 @@ <h1 mat-dialog-title>{{ options.title | translate }}</h1>
     {{ options.cancelText | translate }}
   </button>
   <button
+    *ngIf="options.buttonText"
```

Comment:
I don't think it's necessary. We already have `hideCancel`. The button in the dialog should say something like `Okay` instead of `Cancel`.